examples/contract/contract_balance_query.py

```
@@ -0,0 +1,103 @@
+
+    print("✅Balance retrieved successfully!")
+    print(f"  Contract: {contract_id}")
+    print(f"  Hbars: {balance.hbars}")
+    return balance
+
+
+def main():
+    try:
+        client = setup_client()
+
+        initial_balance_tinybars = Hbar(1).to_tinybars()
```

Ruff (0.14.10)

Local variable `initial_balance_tinybars` is assigned to but never used

Remove assignment to unused variable `initial_balance_tinybars`

(F841)